api/src/wfl/module/covid.clj

```diff
@@ -152,6 +152,22 @@
                              {:workload workload})))
     (if-not (or stopped finished) (stop! workload (utc-now)) workload)))
 
+(defn ^:private retry-covid-workload
+  "Retry/resubmit the `workflows` managed by the `workload` and return the
+   workload that manages the new workflows."
+  [{:keys [finished id executor] :as workload} workflows]
+  (when-not finished
```

Comment:
I did note this specification from the spike doc:

"The /retry request reanimates a workload if it was finished **or stopped**…"

But I elected only to check if the workload had finished, since I don't think we want to allow a retry for a workload that's still running in any capacity.  LMK if this is incorrect.

Reply:
I think it's is right - stopped basically means stop looking for new inputs. It doesn't mean stop processing.